### PR TITLE
upgrade to vty >=6

### PR DESCRIPTION
**Pull request by minimapletinytools**
This PR upgrades reflex-vty to use vty >=6 which has a few breaking changes in order to support cross platform.

Testing: ran examples fine on mac. Still need to try and linux and windows.

### CODE REVIEW

**Comment by minimapletinytools** [comment]
Not sure where bimap should go in default.nix and I may have gotten the other callHackageDirect stuff wrong. `default` is failing but everything else seems to be passing

**Review by ali-abrar — APPROVED** [review]

**Comment by ali-abrar** [comment]
I'll have a look at the failing check and then this can be merged. Thanks!

File: reflex-vty.cabal [hunk 1]
```diff
@@ -40,7 +40,8 @@ library
                  , Reflex.Spider.Orphans
                  , Control.Monad.NodeId
   build-depends:
-    base >= 4.10.0 && < 4.19,
+    base >= 4.10.0 && < 4.20,
+    bimap >= 0.3.3 && < 0.5,
```
Comment (414owen) [inline reflex-vty.cabal:44]: Looks like `bimap` is in `build-depends` twice now?
Does the CI pass if you remove one of the `bimap` lines?

Comment (minimapletinytools) [reply reflex-vty.cabal]: ah good catch, fixed. looks like that was it.

**Comment by ali-abrar** [comment]
Released to hackage as 0.5.2.0, along with #78

**Pull request merged**